app/models/create_expenses_with_receipt.rb

```diff
@@ -14,11 +14,48 @@ def create
     save
   end
 
+  def update(expense)
```

Comment:
@zamith I placed this method in this class for (some) convenience.
Should this be here? If so, should the class be renamed?